docs/_sidebar.md

```diff
@@ -30,8 +37,9 @@
 
 - Troubleshooting
 
+  - [Adding Device Support](https://zwave-js.github.io/node-zwave-js/#/config-files/contributing-files)
+  - [Requesting Device Support](https://github.com/zwave-js/node-zwave-js/issues/new?assignees=&labels=config+%E2%9A%99&template=device_config.yml&title=Missing+device+configuration%3A+%3Center+manufacturer+and+device+name%3E)
   - [Bug Report](troubleshooting/bug_report.md)
```

Comment:
I feel like making driver logs is hidden. We request it so often that it deserves a top-level spot under "Troubleshooting" IMO.